vision/cloud-client/detect/beta_snippets.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Google Cloud Vision API Python Beta Snippets
+
+Example Usage:
+python beta_snippets.py -h
+python beta_snippets.py object-localization INPUT_IMAGE
+python beta_snippets.py object-localization-uri gs://...
+python beta_snippets.py handwritten-ocr INPUT_IMAGE
+python beta_snippets.py handwritten-ocr-uri gs://...
+
+
+For more information, the documentation at
+https://cloud.google.com/vision/docs.
+"""
+
+import argparse
+import io
+
+
+# [START vision_localize_objects]
+def localize_objects(path):
+    """Localize objects in the local image.
+
+    Args:
+    path: The path to the local file.
+    """
+    from google.cloud import vision_v1p3beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with open(path, 'rb') as image_file:
+        content = image_file.read()
+    image = vision.types.Image(content=content)
+
+    objects = client.object_localization(
+        image=image).localized_object_annotations
+
+    print('Number of objects found: {}'.format(len(objects)))
+    for object in objects:
```

Comment:
because `object` is a Pyhton built-in global variable, perhaps use `object_` instead.  the plural `objects` is fine though.